docs/source/gcp/examples/spanner-drop-databases.rst

```diff
@@ -0,0 +1,27 @@
+Spanner - Drop databases
+=========================
```

Comment:
please move these docs to action class doc strings, or include examples on the class doc strings which serve as the reference for actions.